src/binder/binder_context.cpp

```diff
@@ -50,7 +51,7 @@ bool BinderContext::GetColumnPosTuple(
     type::TypeId& value_type) {
   auto db_id = std::get<0>(table_id_tuple);
   auto table_id = std::get<1>(table_id_tuple);
-  auto schema = catalog::Catalog::GetInstance()
+  auto schema = catalog::CatalogStorageManager::GetInstance()
```

Comment:
@apavlo I remembered we had this discussion before. If HaoXiang is able to start the transaction before bind & optimize phase, we should read catalog table(in this case pg_attribute) to retrieve related information instead of going to storage level and get memory object?